### DISCUSSION TITLE
Stitching long jump frames

### DISCUSSION THREAD

**Discussion by sharun-s** [OpenStitching/stitching#88] [Q&A]
I am trying to create a single image that shows different stages of the jump
 
![start-28 3](https://user-images.githubusercontent.com/18413274/201290646-52125d82-1033-4914-b5b0-4a182a464ab9.png)
![start-34 0](https://user-images.githubusercontent.com/18413274/201290714-7cec5ef7-9273-445e-ab91-cadf2095184e.png)
![start-36 4](https://user-images.githubusercontent.com/18413274/201290764-bb653a6f-d1ef-4dd8-abbf-74d5469d91e1.png)
Is it possible to do it with this tool?

**Comment by lukasalexanderweber** [comment]
The first result with default settings looks quite promising:

![result](https://user-images.githubusercontent.com/32765578/201336046-1caa5b36-88d6-45c8-b2e9-f5867cc23d03.jpg)

**Comment by sharun-s** [comment]
Thats awesome...but when I tried with default settings I just got the athlete in the first image but missing from the other but the entire background is captured quite well.
Using just the cli version - python3 stitch.py 1.png 2.png ... 10.png --output out.png I got -
![result](https://user-images.githubusercontent.com/18413274/201343931-86febae6-40e9-43f4-b26a-09926308cff1.jpg)

Do I need to add some other options? Does order of images matter?
Thanks a lot!

**Comment by sharun-s** [comment]
I just realized I needed to crop the right side of all the images(too much unnecessary data there) to control the size of the overlaps between consecutive images. Now things are looking much better. This is such a cool tool. Thanks a lot!
